bin/compare-maps

```diff
@@ -27,6 +25,8 @@ parser.add_option("-d", "--degrees", default=False, action="store_true")
 parser.add_option("-r", "--rankmap", default=False, action="store_true", help="make comparisons with rankmaps instead of probability maps")
 
 parser.add_option("", "--dMAP", default=False, action="store_true", help="compute the angular separation between maximum a posteriori points")
+parser.add_option("", "--dMAP-cm", default=False, action="store_true", help="compute the angular separation between the maximum a posteriori point and the center of mass of the posterior")
```

Comment:
I don't think this option is ever used. Did you mean to add a conditional below?